ecommerceapp/Views/Category Screen/CategoryScreen.swift

```diff
@@ -10,8 +10,8 @@ import SwiftUI
 struct CategoryScreen: View {
     var body: some View {
             NavigationView {
-                VStack(alignment: .leading) {
-                    NavigationBarEdit(title: "main.title".locally(), size: 30, height: 100)
```

Comment:
Bi kontrol edelim.
https://developer.apple.com/documentation/swiftui/migrating-to-new-navigation-types